com.avaloq.tools.ddk.xtext.test.core/src/com/avaloq/tools/ddk/xtext/test/jupiter/AbstractQuickFixTest.java

```diff
@@ -386,7 +386,7 @@ private void assertQuickFixProducesExpectedOutput(final String expectedContent,
     if (ignoreFormatting) {
       MatcherAssert.assertThat(message, actual, IsEqualCompressingWhiteSpace.equalToCompressingWhiteSpace(expected));
     } else {
-      assertEquals(message, expected, actual);
+      assertEquals(expected, actual, message);
```

Comment:
Hi @markdomeng ,

I do not understand this change. Why are we changing the assertEquals without changing the import? Will this not break the `assertEquals`?